report_thesis/src/sections/background/ensemble_learning_models/etr.tex

```diff
@@ -4,16 +4,20 @@ \subsubsection{Extra Trees Regressor (ETR)}
 
 \gls{rf} is an ensemble learning method that improves the accuracy and robustness of decision trees by building multiple trees and combining their predictions.
 Each tree is trained on a random subset of the data using bootstrap sampling, where samples are drawn with replacement, meaning the same sample can be selected multiple times.
+Bagging, or bootstrap aggregating, uses these bootstrap samples to create multiple versions of the training set.
 This introduces variability and reduces overfitting, as some data points may appear multiple times while others may be omitted.
```

Comment:
This introduces variability ... in the (training data available to the) models?

Could clarify that, since we later pose reduced variance as a desirable trait (for reducing overfitting)